### PR TITLE
Feature/1704 lightning

### LINKED ISSUE #1704: Settings Page not working in Lightning

### PR DESCRIPTION
**Pull request by njjc**
# Warning
# Info
- "NPSP Resources" tab has been renamed to "Getting Started" and includes lots of new resources to help get the most out of NPSP.

# Issues
Fixed #1704

### CODE REVIEW

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -2981,22 +2981,6 @@ Once the process is 100% complete, you can safely leave or refresh this page.</v
         <value>This setting lets you map fields from the Recurring Donation record to the Recurring Donation&apos;s child Opportunities.</value>
     </labels>
     <labels>
-        <fullName>stgHelpRDFieldMapOppField</fullName>
```
Comment (davidhabib) [inline src/labels/CustomLabels.labels]: @njjc don't we have to keep these labels in!

**Comment by kbromer** [comment]
This should also resolve #707

**Comment by davidhabib** [comment]
@njjc that would require marking the page as available for mobile, I think.  Nic, did you change this or test this?

**Comment by njjc** [comment]
In the interest of getting today's release out, we're going to delay getting the Get Started tab (as it's now called) sf1 ready. Alicia and I will work on the mobile view and responsive design this sprint. @davidhabib let's merge this when you're ready.

**Comment by njjc** [comment]
@kbromer It works in salesforce1 now, although the design needs a tiny bit of tweaking for narrow devices, which we'll do this sprint.

**Pull request merged**